application-blog-ui/src/main/resources/Blog/RssCode.xml

```diff
@@ -194,7 +194,7 @@
       ## This is just a list of blog entries, which are detailed below.
       #foreach ($entryDoc in $entries)
         #if($xwiki.hasAccessLevel('view', ${entryDoc.fullName}))
-        &lt;rdf:li rdf:resource="$entryDoc.getExternalURL('view', "language=${entryDoc.realLanguage}")" /&gt;
+        &lt;rdf:li rdf:resource="$entryDoc.getExternalURL('view', "language=${entryDoc.realLocal.language}")" /&gt;
```

Comment:
@amottier don't forget this one ;)

Reply:
thanks @tmortagne and I also fix the typo `realLocal` → `realLocale`